wgengine/monitor/monitor.go

```diff
@@ -372,8 +372,16 @@ func (m *Mon) pollWallTime() {
 // came out of sleep.
 func (m *Mon) checkWallTimeAdvanceLocked() {
 	now := wallTime()
-	if now.Sub(m.lastWall) > pollWallTimeInterval*3/2 {
-		m.timeJumped = true
+	switch runtime.GOOS {
+	case "android", "ios":
```

Comment:
can you make a new package-level unexported constant for this so we don't duplicate this set of geese in two places and risk them getting out of sync again?

Reply:
Good idea. Done.